mlflow/environment_variables.py

```diff
@@ -81,6 +81,9 @@ def get(self):
 #: (default: ``None``)
 MLFLOW_TRACKING_URI = _EnvironmentVariable("MLFLOW_TRACKING_URI", str, None)
 
+# Add this near other environment variable definitions
+MLFLOW_LOGGING_LEVEL = _EnvironmentVariable("MLFLOW_LOGGING_LEVEL", str, "INFO")
```

Comment:
I think we can declare this variable as `Literal['DEBUG', 'INFO', 'WARNING', 'ERROR', 'CRITICAL']`?

Reply:
@Flametaa , thanks for the suggestion. I made add the Literal type for MLFLOW_LOGGING_LEVEL.

Reply:
Also, @Flametaa Should I include this PR be included as part of next minor release?